hudi-sync/hudi-sync-common/src/main/java/org/apache/hudi/sync/common/AbstractSyncTool.java

```diff
@@ -41,7 +41,7 @@ public AbstractSyncTool(TypedProperties props, Configuration conf, FileSystem fs
 
   @Deprecated
   public AbstractSyncTool(Properties props, FileSystem fileSystem) {
-    this(new TypedProperties(props), new Configuration(), fileSystem);
+    this(new TypedProperties(props), fileSystem.getConf(), fileSystem);
```

Comment:
@rmahindra123 if user provides an fs, shall we always make use of its conf? Actually not sure why the new API allow passing a different conf rather than use the one from `fileSystem`

Reply:
+1